shared/dev/dumb-sheet/render.desktop.js

```diff
@@ -48,12 +48,7 @@ class DumbSheetRender extends Component<void, Props, any> {
   componentDidMount () {
     // After all child components have loaded (and their autoFocuses, if present, have been triggered)
     // return focus back to the main filter field of the dumb sheet.
-    ReactDOM.findDOMNode(this.refs.filterInput).querySelector('input').focus()
-  }
-
-  componentWillReceiveProps (nextProps: Props) {
-    // FIXME: desktop <Input> element keeps internal state, need to set its value manually
-    this.refs.filterInput.setValue(nextProps.dumbFilter)
+    this.refs.filterInput.focus()
```

Comment:
This is just for our dev sheet so it doesn't get focus stolen from components it houses